src/views/FarmAuction/components/TabToggle.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { Flex } from '@pancakeswap/uikit'
+import styled from 'styled-components'
+
+const Wrapper = styled(Flex)`
+  overflow-x: scroll;
+  padding: 0;
+  border-radius: 24px 24px 0 0;
+  ::-webkit-scrollbar {
+    display: none;
+  }
+  scrollbar-width: none; /* Firefox */
+`
+
+const Inner = styled(Flex)`
+  justify-content: space-between;
+  background-color: ${({ theme }) => theme.colors.input};
+  width: 100%;
+`
+
+interface TabProps {
+  isActive?: boolean
+  onClick?: () => void
+}
+
+export const TabToggle = styled.button<TabProps>`
```

Comment:
The component is shown on screenshot.
There is very similar component on Info site, once we migrate it we can refactor them and put into UIKit.
UIKit has TabMenu component which is kinda similar but quite different in visuals and usage. 
![Screenshot 2021-08-02 at 2 30 40](https://user-images.githubusercontent.com/81824236/127788479-0e7e19df-66c9-4db4-b841-16d1f6bae510.png)

Reply:
put this comment in the code, or we will forget to do it

Reply:
Added comment dac5250d23185a4cb3370f7b086921ac357ba557